public/js/app.js

```diff
@@ -1,106 +1,82 @@
-        Timer.timeStarted = new Date().getTime();
-        var endTime = new Date().setTime(Timer.timeStarted + timeRemaining);
-        $("#timer").countdown(endTime, function(event) {
-            $(this).text(event.strftime('%H:%M:%S'));
-        });
-        // Only play bell if the Timer._countdown function was called due to a swap between
-        // session and break, not if it was called to unpause.
-        if(Timer.paused === false)
-            Timer.bell.play();
-        Timer.paused = false;
-    },
-    _setCountdown: function(lenLocId, breakLabel) {
-        // Set variables
-        var label = $('#circleLabel').text() ;
-        var lenLocId = (label === 'Start' || label === 'Break') ? '#session-len': '#break-len';
-        var breakLabel = (label === 'Start' || label === 'Break') ? 'Session': 'Break';
-        // Set up countdown
-        Timer.timeRemaining = Number($(lenLocId).text()) * Timer.minute;
-        $('#circleLabel').html(breakLabel);
-        Timer._countdown(Timer.timeRemaining);
-        // Set timer for the following countdown
-        Timer.timeTillNext = setTimeout( function() {
-            Timer._setCountdown();
-        }, Timer.timeRemaining);
-    },
-    _reset: function() {
-        $('#circleLabel').html('Start');
-        clearTimeout(Timer.timeTillNext);
-        if(Timer.paused === false)
-            $('#timer').countdown('stop');
-        $('#timer').html('00:00:00');
-        Timer.paused = true;
+app.controller('ctrl', function($scope, $timeout) {
+    // Initialize variables
+    var minute = 60;
+    var paused = true;
+    var bell= document.createElement('audio');
+    bell.setAttribute('src', 'http://soundbible.com/grab.php?id=2084&type=wav');
+
+    $scope.length = {
+        break: 5,
+        session: 25
+    };
+    $scope.labels = {
+        start: 'Start',
+        session: 'Session',
+        break: 'Break'
+    };
+    $scope.timerLabel = $scope.labels.start;
+    $scope.timer = $scope.length.session * minute;
+
+    // Acts as the "control" for the countdown, determining the countdown's behavior
+    // everytime it's clicked on (i.e. starting, pausing,and resuming)
+    $scope.timerControl = function () {
+        // start countdown
+        if($scope.timerLabel === $scope.labels.start && paused === true) {
+            $scope.setUpNextCountdown($scope.length.session * minute, $scope.labels.session);
+        // pause countdown
+        } else if(paused === false) {
+            $timeout.cancel(countdown);
+            paused = true;
+        // resume countdown
+        } else {
+            $scope.countdown();
+            paused = false;
+        }
+    };
+
+    // Re-initializes variables for the next countdown
+    $scope.setUpNextCountdown = function(length, label) {
+        $scope.timer = length;
+        $scope.timerLabel = label;
+        $scope.countdown();
+        bell.play();
+        paused = false;
+    };
+
+    // Function for the actual countdown
```

Comment:
Prob shouldn't use the same 'countdown' name for your scope variable and local variable.